### DISCUSSION TITLE
bumblebee-status icon overlapping

### DISCUSSION THREAD

**Discussion by alexadamm** [i3/i3#5879] [Q&A]
just installed bumblebee-status today, i have a trouble in using bumblebee-status. with this i3 bar config:

```
bar {
        font pango:JetBrainsMono Nerd Font 8.5
        tray_output none
        status_command bumblebee-status -t dracula-powerline \
          -m cpu memory battery load disk nic arch-update dunst datetime 

}
```

i got the icons are overlapping
![image](https://github.com/i3/i3/assets/59623858/a5328d4d-611c-4eab-825f-55dfea77c7dd)

i have adjusted the font style and font size but it wont help

**Comment by mikeredev** [comment]
this is more of a bumblebee-status question but you could try using `ttf-font-awesome` (Font Awesome 6 Free). I always have issues with nerd fonts being misaligned

**Reply by orestisfl** [reply]
I don't think it is, bumblebee-status generates text and i3bar displays it. If it's a graphical error it could be i3bar's problem.

I agree regarding the font change suggestion though.

**Reply by alexadamm** [reply]
i've tried it, changing font styles doesn't help anything

**Reply by sebatihm** [reply]
I had the same problem, in my case i had to unistall some nerd fonts, and it worked
